packages/sit-onyx/src/styles/global.css

```diff
@@ -1,3 +1,5 @@
+@layer onyx.reset, onyx.utility, onyx.component, onyx.density;
+
```

Comment:
Lars and I found a bug - the import in layers.scss itself is not enough, in the alpha test app we noticed that when importing the global.scss, the styles overwrote the reset layer and the margin got cancelled out

Reply:
I can demonstrate it to you in a call, if you want :)

Reply:
The global.css is imported and style.css are imported in "wrong" order…
Can't we import the `layers.scss` here, too? Then we have it defined in a single file, or you need to use a different layer name here

Reply:
Info from Lars to that topic:

> No, because global.css will be imported by the project directly and then there's no relative import path to layers.css